### DISCUSSION TITLE
egui_glow example of using image texture

### DISCUSSION THREAD

**Discussion by d10sfan** [emilk/egui#1577] [Q&A] [answered]
I saw that there was an example of using an image with glium (https://github.com/emilk/egui/blob/master/egui_glium/examples/native_texture.rs), along with examples of how to do it in eframe.

When using glow (using this as a basis - https://github.com/emilk/egui/blob/master/egui_glow/examples/pure_glow.rs), I was unclear how to use an image similar to the example above. I'd like to get a texture_id from importing an image into glow and then egui, for use in a button.

**Comment by d10sfan** [comment, marked as answer]
I just saw the RetainedImage extra, so was able to use that.

**Reply by pauladam94** [reply]
RetainedImage is now deprecated. I am interesting in the answer of this question without RetainedImage.
